### PR TITLE
included graphlb

### PR DESCRIPTION
**Pull request by mettuaditya**
The repository will be updated with more algorithms and data structures as time moves on.

### CODE REVIEW

**Comment by mettuaditya** [comment]
Please do mention any modification/improvements needed to be done in the repository.

File: README.md [hunk 1]
```diff
@@ -83,6 +83,7 @@ Contributions are welcome. Please take a quick look at the [contribution guideli
  * [crystalline](https://github.com/jtomschroeder/crystalline) - A collection of containers and algorithms
  * [delimiter_tree](https://github.com/drujensen/delimiter_tree) - A tree structure that is built using a delimiter
  * [edits.cr](https://github.com/tcrouch/edits.cr) - Collection of edit distance algorithms
+ * [graphlb](https://github.com/mettuaditya/graphlb) - collection of graph datastructure and algorithms
```
Comment (veelenga) [inline README.md]:
```suggestion
 * [graphlb](https://github.com/mettuaditya/graphlb) - Collection of graph datastructure and algorithms
```

**Comment by mettuaditya** [comment]
I have updated the Repo as you have suggested. Please look at the changes and comment if any changes need to be done.

**Comment by veelenga** [comment]
Looks better to me, thank you. Please make a good test coverage (you added only one test).

Also, right now we are not able to merge it because of [a policy we have](https://github.com/veelenga/awesome-crystal/blob/master/.github/CONTRIBUTING.md): 

> If you just created something, wait at least 30 days before submitting to give it some time to mature and ensure it's not just a publish-and-forget type of project

**Comment by mettuaditya** [comment]
I guess now my repository is odd enough to be merged and it satisfies all the policies. Could you please take a look at it and tell me if any changes are required so that I can work on it as soon as possible.

**Comment by veelenga** [comment]
@mettuaditya 

Please correct this:

1. Your CI badge should have a link to the build system. Use this in your Readme:

```
[![Build Status](https://travis-ci.org/mettuaditya/graphlb.svg?branch=master)](https://travis-ci.org/mettuaditya/graphlb)
``` 

2. Did you configure weekly builds?

**Comment by vladfaust** [comment]
My 2 cents here: 

1. Including into the top namespace is generally a bad idea (as seen at https://github.com/mettuaditya/graphlb/blob/master/src/graphlb/algorithms/bellman_ford.cr#L3)
2. Please add a list of currently implemented algorithms to the To Do like this:

## To Do

* Algorithms
  - [ ] Edmonds-karp
  - [ ] Floyd-warshall
  - [x] Bellman-ford
  - [ ] etc.

3. You shard doesn't have [releases](https://github.com/mettuaditya/graphlb/releases) yet
4. "Then run `crystal deps`" is outdated information, `shards install` is commonly used now
5. Please add description to your repository
6. It's a good idea to publish API docs online. It could be done with [Travis GitHub Pages deployment](https://docs.travis-ci.com/user/deployment/pages/). Just don't forget to add these lines to your `.travis.yml`:

```diff
script:
  - crystal spec
+  - crystal docs
+ deploy:
+   provider: pages
+   skip_cleanup: true
+   keep_history: true
+   github_token: $GITHUB_TOKEN
+   on:
+     branch: master
+   local_dir: docs
```

**Comment by mettuaditya** [comment]
I have made the changes as per suggestion please take a look into it and inform me in-case any changes are required.

**Comment by veelenga** [comment]
@vladfaust could you pls ^

**Pull request merged**